### PR TITLE
Style the leaderboard game

### PR DESCRIPTION
**Pull request by Ridwanullahi-code**
- Add customized background to refresh and submit buttons
- Styling the button states.

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -18,42 +18,72 @@ main {
...
+  color: white;
+}
+
+.submit-btn:hover {
+  opacity: 0.8;
+}
+
+.submit-btn:active {
+  background-color: rgb(0, 0, 44);
+}
```
Comment (codecaiine) [inline css/index.css]: - [ ] Good job with your design but please make sure that your design layout looks like the leaderboard [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png) as required for this project as a developer you need to pay attention to details :+1: 

#### Your design
Screenshot from 2022-10-20 20-20-03

#### Wireframe
Screenshot from 2022-10-20 20-25-10

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Ridwanullahi-code , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
- Good design :heavy_check_mark: 

## Required Changes ♻️
- [ ] Please make sure to add a license in your repo root, just create a new license file and copy this [MIT](https://choosealicense.com/licenses/mit/) license and customize it by adding your name to make your readme more professional :smile: 
- [ ] Please add a link to your repo issue in your readme in case someone want to contribute to your project in order to make your readme more professional :heavy_check_mark: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
+            <div class="result">
+            </div>
         </section>
         <section class="add-section">
             <h2>Add your score</h2>
-            <form>
+            <form method="POST">
                 <input type="text" name="name" class="name" placeholder="your name">
                 <input type="text" name="score" class="score" placeholder="your score">
                 <input type="button" name="submit" class="submit-btn" value="submit">
```
Comment (fobadara) [inline dist/index.html:29]: [Required] Kindly ensure that the submit button is aligned to the right. As the previous reviewer suggested you should ensure that your design matches the template.

File: README.md [hunk 3]
```diff
@@ -1,53 +1,76 @@
...
+Contributions, issues, and feature requests are welcome!
 
-
+Feel free to check the [issues page](https://github.com/Ridwanullahi-code/Leaderboard/issues/)
 ## **Acknowledgements**
- - [Microverse](https://www.microverse.org/)
+
+- [Microverse](https://www.microverse.org/)
 
 ## **License**
```
Comment (fobadara) [inline README.md:76]: - Your license does not include your details like fullname.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Ridwanullahi-code, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
👍🏽 Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- You don't need to add. `feature/` before the feature branch's name.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @Ridwanullahi-code,
### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**